### PR TITLE
CT-1191 Add a Trigger marker to case details

### PR DESCRIPTION
**Pull request by aliuk2012**

### CODE REVIEW

File: spec/decorators/case_decorator_spec.rb [hunk 1]
```diff
@@ -125,6 +125,24 @@
     end
   end
 
+  describe '#trigger_case_marker' do
+    context 'unflagged case' do
+      it 'returns space' do
```
Comment (misaka) [inline spec/decorators/case_decorator_spec.rb]: Why does it need to return a space?

Comment (aliuk2012) [reply spec/decorators/case_decorator_spec.rb]: the table cell needs it.  We do this for the draft deadline too https://github.com/ministryofjustice/correspondence_tool_staff/blob/develop/app/decorators/case_decorator.rb#L35

File: spec/decorators/case_decorator_spec.rb [hunk 2]
```diff
@@ -125,6 +125,24 @@
...
+        unflagged_case = create(:case).decorate
+        expect(unflagged_case.trigger_case_marker).to eq ' '
+      end
+    end
+
+    context 'flagged case' do
+      it 'returns the internal deadline' do
```
Comment (misaka) [inline spec/decorators/case_decorator_spec.rb]: Wrong spec here I think, but not worth blocking the PR for it.

Comment (aliuk2012) [reply spec/decorators/case_decorator_spec.rb]: doh, copy and paste

**Pull request merged**